test/topology_custom/test_mv_building.py

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import pytest
+from test.pylib.manager_client import ManagerClient
+
+from test.pylib.util import wait_for_view
+
+# This test makes sure that view building is done mainly in the streaming scheduling group
+# and not the gossip scheduling group. We do that by measuring the time each group was
+# busy during the view building process and confirming that the gossip group was busy
+# much less than the streaming group.
+# Reproduces https://github.com/scylladb/scylladb/issues/21232
+@pytest.mark.asyncio
+async def test_view_building_scheduling_group(manager: ManagerClient):
+    server = await manager.server_add()
+    cql = manager.get_cql()
+    await cql.run_async(f"CREATE KEYSPACE ks WITH replication = {{'class': 'NetworkTopologyStrategy', 'replication_factor': 1}}")
+    await cql.run_async(f"CREATE TABLE ks.tab (p int, c int, PRIMARY KEY (p, c))")
+
+    concurrency = 100
+    async def do_inserts(i: int, max: int):
+        while i < max:
+            await manager.cql.run_async(f"INSERT INTO ks.tab(p, c) VALUES ({i}, {i})")
+            i += concurrency
+
+    await asyncio.gather(*[asyncio.create_task(do_inserts(i, 50000)) for i in range(concurrency)])
```

Comment:
Does this concurrency help? Does a concurrency of 100 help more than, say, 5?

Reply:
It seemed to help but the approach with unlogged batches is much faster anyway